packages/app/src/app/pages/common/Modals/PreferencesModal/Badges/index.tsx

```diff
@@ -1,12 +1,20 @@
-import React from 'react';
-import { inject, observer } from 'app/componentConnectors';
+import React, {FunctionComponent} from 'react';
+import { useOvermind } from 'app/overmind';
 
 import Margin from '@codesandbox/common/lib/components/spacing/Margin';
 import Badge from '@codesandbox/common/lib/utils/badges/Badge';
 import { Title } from '../elements';
 
-function BadgesComponent({ store, signals }) {
-  const badgesCount = store.user.badges.length;
+export const Badges: FunctionComponent = () => {
+  const {
+    state: {
+      user: {badges},
```

Comment:
```suggestion
      user: { badges },
```

eslint --fix should have caught this, not sure why it didn't. Adding spaces for consistency